### PR TITLE
Make config_setting_group visibility aware

### PR DESCRIPTION
**Pull request by kwasimensah**
There might be other attributes of config_setting you want to expose. I didn't want to do a general kwargs since this is using alias underneath and that might allow leaking attributes you don't want the macro to support.

### CODE REVIEW

File: lib/selects.bzl [hunk 1]
```diff
@@ -107,6 +107,7 @@ def _config_setting_group(name, match_any = [], match_all = []):
       match_all: A list of `config_settings`. This group matches if *every*
           member in the list matches. If this is set, `match_any` must be not
           set.
+      visibility: Visibiliity of the main alias used.
```
Comment (gregestren) [inline lib/selects.bzl]: From a user perspective, would it be clearer to call this the visibility of the config_setting_group?

Also pedantic typo nit: double i's in the second visibility!

Comment (kwasimensah) [reply lib/selects.bzl]: typos and comment fixed

**Comment by c-parsons** [comment]
Apologies, I'm a little unclear on how this makes config_setting_group "select aware", I understand that it repropagates visibility, which is a useful feature.

As an aside, I was in a conversation recently with some other interested parties, and we tried to tabulate which attributes we'd like to recommend are almost always propagated by macro definitions, I think we came up with: `visibility`, `tags`, and `testonly`.

**Comment by kwasimensah** [comment]
Meant to say "visbility" updated the title. Can we do the other attributes in a separate pull request? This is blocking me on a migration I'm trying to make and I'd like to get this is in ASAP

**Pull request merged**